source/fab/dep_tree.py

```diff
@@ -27,8 +27,10 @@ class AnalysedFile(object):
     The object can represent itself as a dict for use with a csv.DictWriter.
 
     """
-    def __init__(self, fpath: Union[str, Path], file_hash: int, module_defs=None, symbol_defs=None,
-                 symbol_deps=None, file_deps=None, mo_commented_file_deps=None):
+    def __init__(self, fpath: Union[str, Path], file_hash: int,
+                 module_defs=None, symbol_defs=None,
+                 module_deps=None, symbol_deps=None,
+                 file_deps=None, mo_commented_file_deps=None):
```

Comment:
This argument list is getting quite long. That tends to be a code smell. Do you have ideas on how it can be brought back inn hand?

Reply:
Agreed. We've not considered solutions yet. We've made [a ticket](https://github.com/metomi/fab/projects/8#card-84696461) to come back to this.